src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package duke.task;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class represents a particular type of Task, corresponding to tasks of the "Deadline" form
+ */
+public class Deadline extends Task {
+    private String deadlineString;
+    private LocalDate deadline = null;
```

Comment:
Similarly, would be easier to identify the context of the variable if the name is dateAndTime.